examples/viewer-integr-messaging.js

```diff
@@ -31,18 +31,19 @@ var APP = '__AMPHTML__';
  * https://developer.mozilla.org/en-US/docs/Web/API/Channel_Messaging_API
  */
 class WindowPortEmulator {
-  constructor(messageHandlers, id, port) {
+  constructor(messageHandlers, id, port, opt_isWebview) {
```

Comment:
The actual code below is preferred. There you only use pass `isWebview` to the `Messaging` class. It'd be better here as well.